_faq/coala_will_apply_for_gsoc_2018.md

```diff
@@ -0,0 +1,3 @@
+---
+question: "coala will apply to GSOC 2018"
+---
```

Comment:
this is totally wrong

Reply:
for the question part write
`Will coala apply for GSoC 2018?`

Reply:
for its answer you have to mention the timeline

Reply:
and a positive message
like
> Yes, coala will apply for Google Summer of Code - 2018. You might read the (GSoC Timeline)[https://developers.google.com/open-source/gsoc/timeline] to get an idea of how things will proceed.

Reply:
Oh sorry I forgot to add the timeline there

will do it as you said